src/components/ListOfItems/ListOfItems.tsx

```diff
@@ -0,0 +1,21 @@
+import { v4 as uuidv4 } from 'uuid';
+
+interface Props {
+  currentItems: string[];
+}
+
+export const ListOfItems: React.FC<Props> = props => {
+  const { currentItems } = props;
+
+  return (
+    <ul>
+      {currentItems.map(item => {
+        return (
+          <li key={uuidv4()} data-cy="item">
```

Comment:
Using `uuidv4()` to generate keys for list items is not recommended for static lists because it generates a new key on every render, causing unnecessary re-renders. Consider using a more stable key, such as the item itself or its index, if the items are unique and static.